webservices/filters.py

```diff
@@ -71,6 +71,25 @@ def filter_fulltext(query, kwargs, fields):
                 query = query.filter(sa.or_(*filters))
     return query
 
+def filter_multi_start_with(query, kwargs, fields):
+    for key, column in fields:
+        if kwargs.get(key):
+            exclude_list = [parse_exclude_arg(value) for value in kwargs[key] if is_exclude_arg(value)]
```

Comment:
These 2 lines occur so often. Perhaps they can be extracted into a function?

Reply:
Good point!  We should create a separate ticket as enhancement for this issue.

Reply:
I suggest that as the work required is so small, we just do it rather than create a separate ticket, groom it, assign it in a sprint, etc.